docker/snl-api-db/docker-entrypoint-initdb.d/01-init-users.sql

```diff
@@ -0,0 +1 @@
+/* placeholder for any extensions to users configuration on db */
```

Comment:
check where is this file run, AFAIR it is to setup some users for developers so not on azure, but just locally. put officer1, judge1,admin users and asd pass here for backward compatibility. check if this file is still needed at all.